[2022-08-19 23:11:23,476] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T06:59:00+00:00 [queued]>
[2022-08-19 23:11:23,505] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T06:59:00+00:00 [queued]>
[2022-08-19 23:11:23,507] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:11:23,508] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 23:11:23,511] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:11:23,550] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_telegram_version> on 2020-01-01 06:59:00+00:00
[2022-08-19 23:11:23,559] {standard_task_runner.py:52} INFO - Started process 4533 to run task
[2022-08-19 23:11:23,583] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sefa_dag', 'get_telegram_version', 'scheduled__2020-01-01T06:59:00+00:00', '--job-id', '868', '--raw', '--subdir', 'DAGS_FOLDER/sefa_dag.py', '--cfg-path', '/tmp/tmpdnjdwwn3', '--error-file', '/tmp/tmpaox9wns0']
[2022-08-19 23:11:23,594] {standard_task_runner.py:80} INFO - Job 868: Subtask get_telegram_version
[2022-08-19 23:11:23,790] {task_command.py:371} INFO - Running <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T06:59:00+00:00 [running]> on host fd7a784b9251
[2022-08-19 23:11:24,039] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sefa_dag
AIRFLOW_CTX_TASK_ID=get_telegram_version
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T06:59:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T06:59:00+00:00
[2022-08-19 23:11:24,041] {python.py:173} INFO - Done. Returned value was: 1.24.0
[2022-08-19 23:11:24,104] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=sefa_dag, task_id=get_telegram_version, execution_date=20200101T065900, start_date=20220819T231123, end_date=20220819T231124
[2022-08-19 23:11:24,173] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 23:11:24,220] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
